tools/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project(tools C)
```

Comment:
Why do you create a new project?

Reply:
Normally I always do that. project() gives you some nice variables <project>_SOURCE_DIR and <project>_BINARY_DIR which are quite handy if you have headers in that directory and you need to include them somewhere else. However we don't use it so I can remove it.